backend/tests/core/ServiceHelper.cs

```diff
@@ -58,8 +60,13 @@ public static class ServiceHelper
         /// <returns></returns>
         public static T CreateService<T>(this TestHelper helper, ClaimsPrincipal user, params object[] args) where T : IService
         {
-            var dbName = StringHelper.Generate(10);
-            return helper.CreateService<T>(helper.CreatePimsContext(dbName, user, false), args);
+            if (!helper.Services.Any(s => s.ServiceType == typeof(PimsContext)))
```

Comment:
If there is already a PimsContext in the service provider it will use it instead of creating a new one.  This reduces the amount of boilerplate code in tests.